### ISSUE TITLE
`date.recent` not working as expected.

### ISSUE THREAD

**Issue by nikhilbadyal** [json-schema-faker/json-schema-faker#730]
Data Generated is not recent.
PS - Everything work when I test in [browser](https://json-schema-faker.js.org/). But in local when I do `jsf test.json` it doesn't work. Does web version ship with some default options which are not available in CLI ?

```
{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "$id": "https://example.com/schema.json",
  "title": "Test Logs",
  "description": "Test Logs JSON Schema",
  "type": "object",
  "properties": {
    "created_at": {
      "description": "Date of Error Message",
      "type": "string",
      "format": "date-time",
      "faker": "date.recent"
    }
  },
  "required": [
    "created_at"
  ]
}
```

Expected- 
`{"created_at":"2022-12-01T13:38:01.0Z"}` or some nearby date.

Actual - 
`{"created_at":"1992-07-08T11:31:54.0Z"}`

<img width="458" alt="image" src="https://user-images.githubusercontent.com/59223300/206759344-3b9e7a04-55bd-41f6-a4be-c2c21324fe9e.png">

**Comment by pateketrueke** [comment]
Oh yes, definitely there is a difference: on the web app we're including FakerJS as dependency, but on the CLI we're not!

Definitely that's an issue, right? What am afraid of, is that if we include FakerJS as dependency, then we should include every other dependency that can be used as well?

I don't think that will scale for ever, so I am thinking on a more flexible solution that should work fine with the CLI... once I figure out this I'll be shipping a patch for this use-case, thank you!

**Comment by nikhilbadyal** [comment]
Any hack I can do for now ? 

I need good amount of fake json data. I want to ingest these fake records. I kind of need 4-5 json/3 sec. I thought to simply add an infinite while loop in a bash file.
So copy paste from the Web version isn't really an option.(probably). 

Anything else you can suggest ?

**Comment by nikhilbadyal** [comment]
Thanks for this and also for the quick replies, It worked 🚀

**Comment by pateketrueke** [comment]
Resolved on #735 -- once released, you'll be able to pass a `--config` flag to configure the CLI generator, e.g.

```js
import faker from '@faker-js/faker';

export default ({ JSONSchemaFaker }) => {
  JSONSchemaFaker.extend('faker', () => faker);
};
```
